### ISSUE TITLE
Empty Content and Parsing Error When Aggregating YouTube Channels

### ISSUE THREAD

**Issue by tillcash** [shouya/rss-funnel#86]
Thank you for the software.

I attempted to aggregate content from multiple YouTube channels using the provided code. However, upon inspection in the UI, it appears that there is no rendered content, although RAW and JSON data can be generated successfully.

When using the feed reader [Yarr](https://github.com/nkanaev/yarr), only the content is empty. Additionally, when trying with the WebExtension [rsspreview](https://addons.mozilla.org/en-US/firefox/addon/rsspreview/) to preview RSS feeds in the browser, it throws an error `XML Parsing Error: prefix not bound to a namespace`.

```
endpoints:
  - path: /youtube.xml
    source:
      format: atom
      title: YouTube
      link: https://www.youtube.com
      description: Aggregate content from multiple YouTube channels and exclude YouTube shorts
    filters:
      - merge:
          filters:
            - discard:
              - shorts
          source:
            - https://www.youtube.com/feeds/videos.xml?channel_id=UCiPmhfdCL06cSVTXKabF0Zg
            - https://www.youtube.com/feeds/videos.xml?channel_id=UCueYcgdqos0_PzNOq81zAFg
```
![Screenshot 2024-03-16 at 09-34-54 RSS Funnel Inspector](https://github.com/shouya/rss-funnel/assets/19629778/0960e4b7-2db0-4b4f-9833-859e55a9bc47)
![Screenshot 2024-03-16 at 09-35-09 RSS Funnel Inspector](https://github.com/shouya/rss-funnel/assets/19629778/9c77a933-709e-4ce0-8573-806e33ebf09d)
![Screenshot_20240316_093609](https://github.com/shouya/rss-funnel/assets/19629778/05d74238-f77a-44ff-a1d0-d4874ed981a9)

**Comment by shouya** [comment]
@tillcash thanks for reporting the issue. The bug has been fixed in #90.

**Comment by shouya** [comment]
@tillcash FYI the bug fix has been released in 0.1.2 (see https://github.com/shouya/rss-funnel/releases/tag/0.1.2).

**Comment by shouya** [comment]
Don't worry, I'm glad to learn about any bugs in the app :)

The reason you're still seeing these "shorts" in the feed is because the value "shorts" is present in the `<media:description>` tag, which specified as part of `media` namespace definition rather than a standard title or content field. As such, they are not checked for the `discard` keywords.

![image](https://github.com/shouya/rss-funnel/assets/526598/4eee72cc-4e21-4e9e-8669-2168d9b37e24)

Now I'm aware of the problem, I will find a way to allow discard/keep_only filter to match inside these non-standard fields. Could you please open an new bug report for the issue? Let's leave the current issue for the original bug report.
